packages/edit-site/src/components/page-pages/index.js

```diff
@@ -10,7 +10,6 @@ import { dateI18n, getDate, getSettings } from '@wordpress/date';
 import { privateApis as routerPrivateApis } from '@wordpress/router';
 import { useSelect, useDispatch } from '@wordpress/data';
 import { DataViews } from '@wordpress/dataviews';
-import { ENTER, SPACE } from '@wordpress/keycodes';
```

Comment:
I can no longer access the editor via the keyboard in any page. Once the preview is focused, the ENTER key doesn't trigger the edit canvas mode. SPACE triggers a scroll.

https://github.com/WordPress/gutenberg/assets/583546/b49cd183-0fcc-4633-b790-a1deb03b1508

Interestingly, in `trunk`, this behavior was only working in dataviews-powered pages, as far as I've tested.

Reply:
I remember having to add an extra div to the `PostPreview` editor wrapper to make this work. Perhaps it wasn't never considered before dataviews? If so, it could be a follow-up for every page to add this behavior to the preview area.

Reply:
The PostPreview has been replaced with the regular "Editor"/"Preview" that we use in the site editor for all the site editor pages basically. So there's a chance that the bug you're mentioning has always existed for the "frame" of the site editor.